package.json

```diff
@@ -22,88 +22,74 @@
     "images"
   ],
   "scripts": {
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "link-test-app": "addon-dev link-test-app",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:devdeps": "addon-dev sync-dev-deps --lint",
```

Comment:
It seemed like, the addon had been made a `v2` addon before it was decided (e.g. via https://github.com/embroider-build/addon-blueprint) that workspaces are a good option.

At the moment, I'm thinking that it may be easier to introduce TypeScript _before_ introducing yarn workspace.